### PR TITLE
Implement Waking the Dragon

### PR DESCRIPTION
**Pull request by zacchiro**
WARNING: this is untested because, for the reasons we're discussing on the dev
chat, I haven't yet restored a fully working local deployment environment...

### CODE REVIEW

File: server/game/cards/events/01/wakingthedragon.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        this.action({
+            title: 'Stand a character',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card =>
+                    card.location === 'play area'
+                    && card.getType() === 'character'
+                    && card.isFaction() === 'targaryen'
+                    && card.isUnique()
+                    && card.kneeled
```
Comment (DukeTax) [inline server/game/cards/events/01/wakingthedragon.js]: Waking the Dragon should be able to be played on standing characters (when you just want to return them to hand).

**Comment by zacchiro** [comment]
On Sun, Apr 30, 2017 at 03:54:48AM -0700, DukeTax wrote:
> Waking the Dragon should be able to be played on standing characters
> (when you just want to return them to hand).

Interesting, I never realized that :)

Fixed!

File: server/game/cards/events/01/wakingthedragon.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    setupCardAbilities() {
+        this.action({
+            title: 'Stand a character',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card =>
+                    card.location === 'play area'
+                    && card.getType() === 'character'
+                    && card.isFaction() === 'targaryen'
```
Comment (cryogen) [inline server/game/cards/events/01/wakingthedragon.js]: isFaction('targaryen')

**Comment by zacchiro** [comment]
On Mon, May 01, 2017 at 06:21:41AM -0700, Stuart Walsh wrote:
> +                    && card.isFaction() === 'targaryen'
> isFaction('targaryen')

Of course!

(fixed)

**Pull request merged**